doc/design/linc_permissions.md

```diff
@@ -0,0 +1,52 @@
+# Permissions for S3 Assets in LINC Archive
+
+Collaborators: Aaron Kanzer, Kabilar Gunalan
+
+## Requirements
+
+• As a CONNECTS developer, I shall be able to view multiple datasets/layers (e.g. dMRI, tractography, HiP-CT) in the same neuroglancer
+• As a CONNECTS developer, I shall be able to view data locally or on neuroglancer
+• As a CONNECTS developer, I shall be able to have direct access to S3 asset URI
```

Comment:
My suggestions update the phrasing and change the order to signify the priority.

```suggestion
1. As a LINC developer, I shall be able to view multiple assets (e.g. dMRI, tractography, HiP-CT) in a single Neuroglancer viewer as layers.
2. As a LINC developer, I shall be able to implement Requirement 1 when Neuroglancer is hosted locally or in the cloud.
```

Reply:
I moved the `direct access to S3 asset URI` requirement to a new section (i.e. `Implementation details`) as this would not be a user requirement, but rather a feature that would need to be implemented to satisfy Requirement 1.